packages/nodejs/src/config.ts

```diff
@@ -120,6 +120,22 @@ export class Configuration {
       ignoreNamespaces: [],
       log: "file",
       logPath: this._tmpdir(),
+      requestHeaders: [
+        "accept",
+        "accept-charset",
+        "accept-encoding",
+        "accept-language",
+        "cache-control",
+        "connection",
+        "content-length",
+        "path-info",
+        "range",
+        "request-method",
+        "request-uri",
+        "server-name",
+        "server-port",
+        "server-protocol"
```

Comment:
```suggestion
        "accept",
        "accept-charset",
        "accept-encoding",
        "accept-language",
        "cache-control",
        "connection",
        "content-length",
        "range"
```

I can't actually find the headers I removed in these two sources:

- https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers
- https://en.wikipedia.org/wiki/List_of_HTTP_header_fields

Might they be legacy fields from the Ruby gem's Rack "environment" values? Or are some of these names custom headers passed along by a proxy server?

Reply:
I noticed this because the list of headers included in the sample is quite short: 
<img width="987" alt="image" src="https://user-images.githubusercontent.com/282402/143014865-9978a092-8273-44f5-8607-198f546a3edb.png">

Reply:
They're from the Elixir integration. Removing them.

Reply:
We'll probably need to remove them from the Elixir integration as well. Because I think you're right, they look like legacy values from the Ruby gem.

Reply:
Ok! Creating an issue.